apps/website/src/content/docs/skiptocontentlink.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: Skip to content link
+description: A skip-to-content link can be used to navigate directly to certain desired contents.
+thumbnail: #TODO
+---
+
+<p>{frontmatter.description}</p>
+
+<Placeholder componentPageName='table--basic' />
+
+## Usage
+
+`SkipToContentLink` is designed for screen reader and keyboard users and will only become visible when focused. It provides a shortcut to the main content of the page, bypassing the header and other navigation elements. The `href` prop should be set to the `id` of the main content section, and the value must include the '#' prefix.
+
+<LiveExample src='SkipToContentLink.main.jsx' truncate={true}>
+  <AllExamples.SkipToContentLinkMainExample client:load />
+</LiveExample>
```

Comment:
If the link is not working anyway, can we simplify the examples? The main purpose of the demo would be to reveal the link upon tabbing.

The instructions can be placed outside the demo if needed.

Also, I think we can make use of static code blocks to show what the correct usage might look like within the context of an application.

Reply:
Hmm, I added a code block under the description and modified the examples. Hmmm, those are just my idea for the examples, I'm not sure if they look good.

Reply:
It doesn't feel quite right; the wording, instructions and code samples do not accurately convey what a skip-to-content link is and how to use it.

I'll make some concrete suggestions tomorrow, but in the meantime I would suggest reading up on skip links in general and within the context of other design systems. That will help you understand what it is and how it should be documented.

- https://webaim.org/techniques/skipnav/
- https://benmyers.dev/blog/skip-links/
- https://css-tricks.com/a-deep-dive-on-skipping-to-content/
- https://design-system.service.gov.uk/components/skip-link/
- https://www.skyscanner.design/latest/components/skip-link/web-xYpbV1AB
- https://ebay.gitbook.io/mindpatterns/navigation/skip-navigation

Reply:
I have tried moving things around and changed the examples in the recent commits.

Reply:
It looks much better now! I'll leave some suggestions separately.